src/components/registration/steps.jsx

```diff
@@ -280,6 +282,17 @@ module.exports = {
         },
         render: function () {
             var formatMessage = this.props.intl.formatMessage;
+            var countryOptions = countryData.countryOptions.concat({
+                label: formatMessage({id: 'teacherRegistration.selectCountry'}),
+                disabled: true,
+                selected: true
+            }).sort(function (a, b) {
+                if (a.disabled) return -1;
+                if (b.disabled) return 1;
+                if (a.value === this.props.defaultCountry) return -1;
+                if (b.value === this.props.defaultCountry) return 1;
+                return 0;
+            }.bind(this));
```

Comment:
I believe it would work to just use `<FormattedMessage>`. Or you may be able to just import `formatMessage` from react-intl? Not sure if that function can be used standalone.